tms-gRPC-Service/deployment.yml

```diff
@@ -0,0 +1,12 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: http://20.90.137.195:8080
+functions:
+  off-tms-grpc-service:
+    namespace: frm-meshed
+    lang: dockerfile
+    handler: .
+    image: frmacr.azurecr.io/tms-grpc-service:latest
```

Comment:
I don't think -grpc- is necessary in the service name. just stick to transaction-monitoring-service instead of tms-service (tmServiceService)

Reply:
Changed ✅ to tms-service